draft-ietf-rats-corim.md

```diff
@@ -2448,8 +2449,27 @@ The comparison MUST return false if there are no hash algorithms from the condit
 
 ##### Comparison for raw-value entries
 
+A `raw-value` entry contains binary data whose meaning the Verifier is not required to understand.
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/71
+The value stored under `measurement-values-map` key 4 in an ACS entry must be a `raw-value` entry, which must have type `bytes`, with an optional tag.
+
+The value stored under the condition ECT `measurement-values-map` key 4 may additionally be a `raw-value-compare` entry, which specifies an expected value and an optional mask.
```

Comment:
```suggestion
```suggestion
The value stored under `measurement-values-map` key 4 in an ACS entry MUST have type `bytes` with an optional tag, or type `raw-value-compare`.
```
```

Reply:
I think it is worthwhile for the spec to call out the difference between codepoint 4 in Evidence/Endorsements and in reference values (eg in stateful-environment-map).